src/installer/corehost/build.sh

```diff
@@ -88,9 +88,10 @@ __LogsDir="$__RootBinDir/log"
 __MsbuildDebugLogsDir="$__LogsDir/MsbuildDebugLogs"
 
 # Set the remaining variables based upon the determined build configuration
+__ConfigTriplet="$(echo $__TargetOS-$__BuildArch |  tr '[:upper:]' '[:lower:]').$__BuildType"
 __DistroRidLower="$(echo $__DistroRid | tr '[:upper:]' '[:lower:]')"
```

Comment:
nit: `__DistroRidLower` is unused with this change.

Reply:
Side note: it would be great if we do not perform this transformation at all in installer partition, and instead use the same approach as other partitions (keep the original casing). The benefit (besides consistency) is that when there is a multibyte character in runtime clone path (e.g. `/Projects/regular_text-ⒾⓇⓇⒺⒼⓊⓁⒶⓇ␣ⓉⒺⓍⓉ/runtime`), the corresponding C++ code in corehost does not perform the correct transformation as `tr(1)`, our PAL and managed code do. The STD C++ code, which corehost is using, only transforms single byte characters in path.

Reply:
I completely agree.